plugins/flutter_aepconsent/pubspec.yaml

```diff
@@ -0,0 +1,37 @@
+name: flutter_aepconsent
+
+description: Official Adobe Experience Platform support for Flutter apps. AEPConsent is a new, innovative product from Adobe to help you easily validate SDK implementations.
```

Comment:
this seems inaccurate, maybe rephrase to something like
Official Adobe Experience Platform support for Flutter apps. The AEP Consent Collection plugin enables consent preferences collection from your Flutter app when using the [Adobe Experience Platform Mobile SDK](https://github.com/Adobe-Marketing-Cloud/acp-sdks) and the Edge Network extension.